test/e2e/kubectl.go

```diff
@@ -498,25 +498,29 @@ var _ = framework.KubeDescribe("Kubectl client", func() {
 			By("checking the result")
 			forEachReplicationController(c, ns, "app", "redis", validateReplicationControllerConfiguration)
 		})
-		It("should reuse nodePort when apply to an existing SVC", func() {
+		It("should reuse port when apply to an existing SVC", func() {
+			// This test doesn't do anything meaningful.  It was supposed to test that the nodeport
```

Comment:
It is intended to do something meaningful - but was written in a way that didn't.  I could just delete the whole test, but ideally we would fix it to test the right thing.